common/fmi2_headers/fmi2FunctionTypes.h

```diff
@@ -0,0 +1,271 @@
+#ifndef fmi2FunctionTypes_h

```

Comment:
These are ver specific to NuRV and lools reusable across all NuRV FMU creation. Probably in `common/models/NuRV`?

Reply:
Those are the standard FMI2 headers, they are not NuRV specific. I think they can stay in `common`